### PR TITLE
PMM-8426 MongoDB backup and section restructure (fresh PR)

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Preview: https://60f03413af73340008eeab8e--pmm-doc.netlify.app/using/backup.html

### CODE REVIEW

File: docs/using/backup.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+
+!!! summary alert alert-info "Summary"
+    - Enable backup features.
+    - Add a storage location.
+    - Satisfy preconditions:
+        - For [MySQL](#mysql-backup-preconditions):
+            - Confirm instance service parameters and storage location.
+            - Install required packages.
+        - For [MongoDB](#mongodb-backup-preconditions):
+            - Install [Percona Backup for MongoDB] on every node in the replica set.
```
Comment (artemgavrilov) [inline docs/using/backup.md:14]: PMM expects that pbm-agent started on each node. So better to make it clear that pbm-agent should be started by the user. 
```suggestion
            - Install and run [Percona Backup for MongoDB] on every node in the replica set.
```

File: docs/using/backup.md [hunk 2]
```diff
@@ -57,46 +46,75 @@
...
+
+        - [`xbstream`][PERCONA_XBSTREAM];
+
+    - [`qpress`][PERCONA_QPRESS].
+
+    **The versions of each must be compatible with the installed version of MySQL.**
+
+## MongoDB backup preconditions
+
+- [Percona Backup for MongoDB] is installed and the agent is running on all MongoDB nodes in the replica set.
```
Comment (artemgavrilov) [inline docs/using/backup.md:90]:
```suggestion
- [Percona Backup for MongoDB] is installed and the pbm-agent is running on all MongoDB nodes in the replica set.
```

File: docs/using/backup.md [hunk 3]
```diff
@@ -57,46 +46,75 @@
...
-    - *Secret Key:*
 
 4. Click *Add* to add the location or *Test* to test the connection.
 
-## Creating a backup
+## MySQL backup preconditions
 
-!!! hint alert alert-success "Tip"
-    You must [add a backup storage location](#adding-a-storage-location) before backing up.
+- There is only one MySQL instance running on the node.
```
Comment (artemgavrilov) [inline docs/using/backup.md:62]: I think we also should mention that `pmm-agent` should run on the same node with mySQL.

File: docs/using/backup.md [hunk 4]
```diff
@@ -57,46 +46,75 @@
...
+
+    - as a service via `systemd`;
+
+    - with the name `mysql` (to confirm, use `systemctl status mysql`);
+
+    - from a `mysql` system user account.
+
+- There is a `mysql` system group.
+
+- MySQL is using the `/var/lib/mysql` directory for database storage.
```
Comment (artemgavrilov) [inline docs/using/backup.md:74]: Also `pmm-agent` should have write permissions to this dir. 

@oter am I right?

File: docs/using/backup.md (hunk 3 above)
Comment (PaulJacobs-percona) [reply docs/using/backup.md:62]:
```suggestion
- [PMM Client](../setting-up/client/index.md) is installed and running on the node.

- There is only one MySQL instance running on the node.
```

File: docs/using/backup.md (hunk 4 above)
Comment (oter) [reply docs/using/backup.md:74]: yes, you are right

File: docs/using/backup.md (hunk 3 above)
Comment (artemgavrilov) [reply docs/using/backup.md:62]: Nice 👍

File: docs/using/backup.md (hunk 4 above)
Comment (PaulJacobs-percona) [reply docs/using/backup.md:76]:
```suggestion
- MySQL is using the `/var/lib/mysql` directory for database storage.

- `pmm-agent` has read/write permissions to the `/var/lib/mysql` directory.
```

**Pull request merged**